hazelcast/src/main/java/com/hazelcast/map/impl/operation/DefaultMapOperationProvider.java

```diff
@@ -215,8 +215,9 @@ public MapOperation createGetOperation(String name, Data dataKey) {
     }
 
     @Override
-    public MapOperation createLoadAllOperation(String name, List<Data> keys, boolean replaceExistingValues) {
-        return new LoadAllOperation(name, keys, replaceExistingValues);
+    public MapOperation createLoadAllOperation(String name, List<Data> keys, boolean replaceExistingValues,
+                                               boolean localOnlyLoad) {
+        return new LoadAllOperation(name, keys, replaceExistingValues, localOnlyLoad);
```

Comment:
maybe we can use `withUserSuppliedKeys` instead of `localOnlyLoad`. `localOnlyLoad` seemed not clear